app/scripts/lib/metrics.js

```diff
@@ -75,6 +75,19 @@ define(function (require, exports, module) {
     }, []);
   }
 
+  function marshallFlowEvent (event, viewName) {
+    if (! viewName) {
+      return `flow.${event}`;
+    }
+
+    if (viewName.indexOf('oauth.') === 0) {
```

Comment:
I have to admit I always have to look up [substr's function signature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr), is it easier to just do:

```js
viewName = /^oauth\./.replace('');
```